alphadia/cli.py

```diff
@@ -189,28 +217,41 @@ def _get_raw_path_list_from_args_and_config(
 
 
 def run(*args, **kwargs):
-    # parse command line arguments
     args, unknown = parser.parse_known_args()
 
     if unknown:
         print(f"Unknown arguments: {unknown}")
         parser.print_help()
-        return
+        return EXIT_CODE_WRONG_CLI_PARAM
 
     if args.version:
-        print(f"{alphadia.__version__}")
+        print(f"{__version__}")
+        return
+
+    # load modules only here to speed up -v and -h commands
+    from alphadia.exceptions import CustomError
+    from alphadia.search_plan import SearchPlan
+    from alphadia.workflow import reporting
+
+    if args.check:
+        print(
+            f"{__version__}"
```

Comment:
I would suggest to return a json here:
`{"version: "1.10.3", "import_success": True}`

Can we wrap the imports in a try except so that we could issue on failure:
`{"version: "1.10.3", "import_success": False}`